src/runtime/manifest-parser.peg

```diff
@@ -1436,8 +1436,14 @@ PrimaryExpression
     const operator = op[0];
     return toAstNode<AstNode.UnaryExpressionNode>({kind: 'unary-expression-node', expr, operator});
   }
-  / NumberType
-  / fieldName
+  / num: NumberType
```

Comment:
For your testing bool will likely be helpful. Might be worth adding it now rather than later.